roles/zuul/files/jobs/hoist.yaml

```diff
@@ -4,15 +4,16 @@
 
     builders:
       - zuul-git-prep
+      - install-ansible
       - shell: |
-          tmpdir=$(mktemp -d)
-          trap "rm -rf $tmpdir" EXIT
-          virtualenv $tmpdir/venv
-          source $tmpdir/venv/bin/activate
-          pip install ansible
-          pip install -r requirements.txt
           ansible-playbook -i inventory/nodepool.py install-ci.yml --skip-tags monitoring -e @secrets.yml.example
           ansible-playbook -i inventory/nodepool.py  tests/validate-ci.yml
 
     publishers:
       - console-log
+
+- project:
+    name: BonnyCI/hoist
+    jobs:
+      - ansible-syntax-{name}:
+        files: "bastion.yml install-ci.yml provision.yml tests/validate-ci.yml"
```

Comment:
Yeah, it would be nice to use a yaml sequence here, but these values are simply passed through to format() in python, and I don't know if we can change how that's interpreted.